serlib/plugins/ltac2_ltac1/dune

```diff
@@ -0,0 +1,12 @@
+(library
+ (name serlib_btauto)
```

Comment:
Wrong name? I'm seeing 
~~~
+ dune build --root . --only-packages=coq-lsp @install
Error: Library serlib_btauto is defined twice:
- serlib/plugins/ltac2_ltac1/dune:3
- serlib/plugins/btauto/dune:3
~~~
in coq CI (eg https://gitlab.inria.fr/coq/coq/-/jobs/4748797)

Reply:
I didn't see this locally, I guess newer dune are more lazy in reporting conflicts, will fix ASAP thanks!

Reply:
Done.